.travis.yml

```diff
@@ -1,8 +1,14 @@
 language: go
+sudo: false
 go:
   - 1.3
```

Comment:
I removed everything prior to go 1.6 and I fixed the compile tester in the shell script which is much neater!